### DISCUSSION TITLE
DreamBooth checkpoint saving issues

### DISCUSSION THREAD

**Discussion by akolstoe** [TheLastBen/fast-stable-diffusion#1068] [Q&A] [answered]
Hopefully, I'm not just missing something, but I haven't been able to identify the issue. When training with DreamBooth on colab with checkpoint saving set at 500 steps, more often than not it gives me the "Converting to CKPT ..." printout, but then instead of saving the checkpoint, it throws "Killed" before "Done, resuming training..." and doesn't save anything that round.

Here's a shot of my run that just finished up. I wasn't able to grab a new one before it cleared everything, but it ended up saving both the 3000 step checkpoint and the 3000 step final checkpoint.

![converting_killed](https://user-images.githubusercontent.com/26512089/208270468-2b52f799-ff7b-4202-aac6-0f82a6c54488.png)

I truly appreciate you making this so easy and available, and thank you pre-emptively for any help!

**Comment by TheLastBen** [comment, marked as answer]
not enough RAM with free colab, but the final checkpoint will be saved successfully
